### PR TITLE
fixed docs

### LINKED ISSUE #267: Docs not present for Ricci Tensor & Vacuum Metrics
**Possible solutions**
  - Additions in `docs/source/api/symbolic/symbolic_index.rst`
  - New files `docs/source/api/symbolic/ricci.rst`, `docs/source/api/symbolic/vacuum_metric.rst`

  - Add `vacuum_metric` to `symbolic/__init__.py`

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by OVSofia**
Fixes #267

### CODE REVIEW

File: docs/source/api/symbolic/christoffel.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Christoffel Symbols Module
 ==========================
 
 This module contains the class for obtaining Christoffel Symbols 
-related to a Metric belonging to any arbitary space-time symbolically:
+related to a Metric belonging to any arbitrary space-time symbolically:
```
Comment (ritzvik) [inline docs/source/api/symbolic/christoffel.rst]: Great, for fixing up the typos :)

**Comment by shreyasbapat** [comment]
@OVSofia the Docs CI is failing! 

Here is why: https://circleci.com/gh/einsteinpy/einsteinpy/3664?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: docs/source/api/symbolic/vacuum_metrics.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Vacuum Metrics Tensor Module
```
Comment (ritzvik) [inline docs/source/api/symbolic/vacuum_metrics.rst]:
```
Vacuum Solutions Module
=======================
```

File: docs/source/api/symbolic/vacuum_metrics.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+=============================
+
+This module contains the definition Schwarzschild Metric
```
Comment (ritzvik) [inline docs/source/api/symbolic/vacuum_metrics.rst]:
```
This module contains various exact vacuum solutions to 
Einstein's Field Equation in form of metric tensor:
```

File: docs/source/api/symbolic/symbolic_index.rst [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
 
 .. toctree::
     :maxdepth: 2
 
-    metric
     christoffel
+    metric
+    ricci
     riemann
+    tensor
```
Comment (ritzvik) [inline docs/source/api/symbolic/symbolic_index.rst]: Remove tensor from here, as the corresponding `.rst` file is not there. It is meant to be used as a base class not not meant to be exposed.

**Comment by OVSofia** [comment]
> @OVSofia the Docs CI is failing!
> 
> Here is why: https://circleci.com/gh/einsteinpy/einsteinpy/3664?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Okey, give me 5 min to fix it

File: docs/source/api/symbolic/symbolic_index.rst (hunk 4 above)
Comment (ritzvik) [reply docs/source/api/symbolic/symbolic_index.rst]: > @OVSofia the Docs CI is failing!
> 
> Here is why: https://circleci.com/gh/einsteinpy/einsteinpy/3664?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

This is why it's failing

**Comment by ritzvik** [comment]
Closing this as it is being superseded by #269

**Pull request closed without merging**